app/src/main/java/com/nexters/zzallang/harusal2/usecase/BudgetUseCase.kt

```diff
@@ -0,0 +1,40 @@
+package com.nexters.zzallang.harusal2.usecase
+
+import android.content.Context
+import android.content.SharedPreferences
+import com.nexters.zzallang.harusal2.application.App
+import com.nexters.zzallang.harusal2.application.util.Constants
+
+
+class BudgetUseCase {
+    private val sharedPreferences: SharedPreferences = App.app.applicationContext.getSharedPreferences(Constants.PREF_NAME, Context.MODE_PRIVATE)
+    private val prefEditor: SharedPreferences.Editor = sharedPreferences.edit()
+
+    fun setAmount(amount: Long){
+        prefEditor.putLong("amount", amount).apply()
```

Comment:
amount도 static으로 빼는게 괜찮을 듯
```suggestion
        prefEditor.putLong(Constants.AMOUNT, amount).apply()
```